udata/templates/search.html

```diff
@@ -16,31 +16,29 @@
 {% set datasets_search_url = url_for('datasets.list') %}
 {% set reuses_search_url = url_for('reuses.list') %}
 {% set organizations_search_url = url_for('organizations.list') %}
+{% set breadcrum_class='search-bar'%}
 
 {% block breadcrumb_bar %}
 <!-- Nav tabs -->
-<ul class="nav nav-tabs search-tabs">
+<ul class="nav nav-tabs search-tabs nav-tabs--search-tabs">
     {% if datasets %}
         <li class="active" data-tab="datasets">
             <a href @click.prevent="setTab('datasets')">
-                {{ _('Datasets')}}
-                <span class="badge">{{ datasets.total }}</span>
+                {{ _('%d Datasets' % datasets.total)}}
```

Comment:
Yes, you can (should) use `ngettext` which allows to have singular/plural support and write:
```html+jinja
{{ ngettext('%(num)d dataset', '%(num)d datasets', datasets.total) }}
```
See http://jinja.pocoo.org/docs/2.9/extensions/#newstyle-gettext for details